js/webcomponent/api-updates-subscribe/main.js

```diff
@@ -141,6 +150,15 @@ class ApiSubscribe extends HTMLElement {
           "Invalid email format!"
         )
       }
+
+      // Log to SiteImprove when clicking Subscribe
+      if(typeof SiteImprove !== 'undefined' && (apiAreaRadios[0].checked || someSelected)){
+        SiteImprove.registerEvent({
+          category: 'Task',
+          action: 'Subscribe to API updates',
+          label: 'Subscribe to API updates'
```

Comment:
A suggestion after discussions:

```suggestion
          label: 'Confirm subscription'
```

Reply:
What about "Confirm email subscription" to also include email? Or is it not necessary 🤔

Reply:
I think we‘ve concluded that it is sufficient with "Confirm subscription". 😄